core_lib/movieexporter.cpp

```diff
@@ -90,6 +90,16 @@ QString ffmpegLocation()
 #elif __APPLE__
     return QApplication::applicationDirPath() + "/plugins/ffmpeg";
 #else
+    QString ffmpegPath = QStandardPaths::findExecutable(
+        "ffmpeg",
+        QStringList()
+            << QApplication::applicationDirPath() + "/plugins"
+            << QApplication::applicationDirPath() + "/../plugins" // linuxdeployqt in FHS-like mode
+    );
```

Comment:
I’m not sure here about the multiline syntax, so if a specific style has already been adopted elsewhere and I just couldn’t find it let me know.